pandas/tests/extension/test_string.py

```diff
@@ -44,32 +54,49 @@ def na_value():
 
 
 @pytest.fixture
-def data_for_grouping():
-    return StringArray._from_sequence(["B", "B", pd.NA, pd.NA, "A", "A", "B", "C"])
+def data_for_grouping(dtype):
+    return dtype.construct_array_type()._from_sequence(
+        ["B", "B", pd.NA, pd.NA, "A", "A", "B", "C"]
+    )
 
 
 class TestDtype(base.BaseDtypeTests):
     pass
 
 
 class TestInterface(base.BaseInterfaceTests):
-    pass
+    def test_view(self, data, dtype, request):
+        if isinstance(dtype, ArrowStringDtype):
+            reason = "Fails until implement, remove before merge"
```

Comment:
I don't fully understand this comment. I don't think we will be able to remove those xfails on the short term?

Reply:
ok, if happy to move forward without the view related tests passing, can change message simply `not implemented`

Reply:
I don't think there will be any way to implement in the foreseeable future (it's not something that is "just" left for a follow-up or so). Arrow inherently cannot modify a string array inplace (because of storing it all in contiguous memory), only if the string that is set is exactly of the same length as the one it is replacing, it could in theory work.

Reply:
unrelated, but should these expectations about views be removed from the EA interface?

Reply:
> unrelated, but should these expectations about views be removed from the EA interface?

That's something we should maybe reconsider, yes. In general the concept of "view" is rather implementation dependent.